lib/spacesuit/api_message.ex

```diff
@@ -5,7 +5,7 @@
 # Just does JSON for now
 defmodule Spacesuit.ApiMessage do
   @derive [Poison.Encoder]
-  defstruct [:status, :message]
+  defstruct [:errorCode, :errorMessage]
```

Comment:
Ok, that's not very compatible with an open source project since no one else is going to be using those. I'll need to think through how we should adapt that.